### PR TITLE
feat: add macOS Catalina deprecation notice

### PR DESCRIPTION
**Pull request by kruskall**
## Motivation/summary

Closes https://github.com/elastic/apm-server/issues/9719

## Checklist

- [ ] Update [CHANGELOG.asciidoc](https://github.com/elastic/apm-server/blob/main/CHANGELOG.asciidoc)
- [ ] Update [package changelog.yml](https://github.com/elastic/apm-server/blob/main/apmpackage/apm/changelog.yml) (only if changes to `apmpackage` have been made)
- [ ] Documentation has been updated

For functional changes, consider:
- Is it observable through the addition of either **logging** or **metrics**?
- Is its use being published in **telemetry** to enable product improvement?
- Have system tests been added to avoid regression?

### CODE REVIEW

File: internal/beatcmd/beat.go [hunk 1]
```diff
@@ -266,6 +267,20 @@ func (b *Beat) Run(ctx context.Context) error {
 		)
 	}
 
+	if runtime.GOOS == "darwin" {
+		v, err := kernelVersion()
+		if err != nil {
+			logger.Warnf("failed to retrieve kernel version, ignoring potential deprecation warning: %v", err)
+		}
```
Comment (axw) [inline internal/beatcmd/beat.go:274]: Should we use `go-sysinfo`?

```go
if host, err := sysinfo.Host(); err != nil {
	logger.Warnf("failed to retrieve kernel version, ignoring potential deprecation warning: %v", err)
} else if host.Info().OS.Major == 19 {
	// macOS 10.15.x (catalina) means darwin kernel 19.y
	...
}
```

Then there's no need for `kernel.go`. Generally I'd prefer to keep platform-specific code out of apm-server, and isolated in libraries like go-sysinfo.

Comment (kruskall) [reply internal/beatcmd/beat.go:274]: I've looked at the go-sysinfo implementation and it does not rely on CGO. However it is reading from a file (https://github.com/elastic/go-sysinfo/blob/main/providers/darwin/os.go#L39), is that a concern ? 

___

The other issue I'm having is with the results: I'm testing the changes on github with the following action:

```
name: GitHub Actions Demo
on: [push]
jobs:
  build:
    runs-on: macos-12
    steps:
      - uses: actions/checkout@v2
      - uses: actions/setup-go@v3.5.0
        with:
          go-version: 1.19
      - name: Run
        run: CGO_ENABLED=0 go run main.go
```

The syscall version is returning `21.6.0` which is mapped to macOS 12.5 (consistently with the github action target).

On the other hand, go-sysinfo returns:
```
&{macos darwin darwin Mac OS X 10.16 10 16 0 21G419 }
Major: 10
Minor: 16
```

That's unexpected. Am I missing something ?

Comment (kruskall) [reply internal/beatcmd/beat.go]: UPDATE:

I looked at the release page (https://opensource.apple.com/releases/): macOS 10.16 doesn't even exist :cold_sweat: 

`host.Info().KernelVersion` seems to be consistent with the custom syscall and the version specified in the github action. It does not rely on CGO so it should be fine (https://github.com/elastic/go-sysinfo/blob/main/providers/darwin/kernel_darwin.go)

Comment (axw) [reply internal/beatcmd/beat.go]: https://eclecticlight.co/2020/08/13/macos-version-numbering-isnt-so-simple/
weird :)

Anyway, using `host.Info().KernelVersion` looks good.

**Review by axw — APPROVED** [review]
LGTM, thanks!

**Comment by marclop** [comment]
I have verified that the deprecation notice isn't logged on my Mac M1 running MacOS Ventura:

<details><summary>Details</summary>
<p>

./apm-server -E output.elasticsearch.username=admin -E output.elasticsearch.password=changeme -E apm-server.expvar.enabled=true -d '*' -e
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.146-0700","log.origin":{"file.name":"beatcmd/beat.go","file.line":139},"message":"Home path: [/Users/marclop/repos/elastic/apm-server] Config path: [/Users/marclop/repos/elastic/apm-server] Data path: [/Users/marclop/repos/elastic/apm-server/data] Logs path: [/Users/marclop/repos/elastic/apm-server/logs]","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"debug","@timestamp":"2023-05-09T09:19:54.147-0700","log.logger":"beat","log.origin":{"file.name":"beatcmd/beat.go","file.line":172},"message":"Beat metadata path: /Users/marclop/repos/elastic/apm-server/data/meta.json","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.147-0700","log.origin":{"file.name":"beatcmd/beat.go","file.line":146},"message":"Beat ID: bf2a6d26-c5ed-4b41-a507-feab5b75c095","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.148-0700","log.logger":"beat","log.origin":{"file.name":"beatcmd/beat.go","file.line":576},"message":"Beat info","service.name":"apm-server","system_info":{"beat":{"path":{"config":"/Users/marclop/repos/elastic/apm-server","data":"/Users/marclop/repos/elastic/apm-server/data","home":"/Users/marclop/repos/elastic/apm-server","logs":"/Users/marclop/repos/elastic/apm-server/logs"},"type":"apm-server","uuid":"bf2a6d26-c5ed-4b41-a507-feab5b75c095"},"ecs.version":"1.6.0"}}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.148-0700","log.logger":"beat","log.origin":{"file.name":"beatcmd/beat.go","file.line":584},"message":"Build info","service.name":"apm-server","system_info":{"build":{"commit":"3503f66af13c5ef1c4439a37584aacc1751464c5","time":"2023-05-09T15:18:59.000Z","version":"8.9.0"},"ecs.version":"1.6.0"}}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.148-0700","log.logger":"beat","log.origin":{"file.name":"beatcmd/beat.go","file.line":587},"message":"Go runtime info","service.name":"apm-server","system_info":{"go":{"os":"darwin","arch":"arm64","max_procs":8,"version":"go1.20.4"},"ecs.version":"1.6.0"}}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.148-0700","log.origin":{"file.name":"beatcmd/maxprocs.go","file.line":68},"message":"maxprocs: Leaving GOMAXPROCS=8: CPU quota undefined","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.148-0700","log.logger":"beat","log.origin":{"file.name":"beatcmd/beat.go","file.line":591},"message":"Host info","service.name":"apm-server","system_info":{"host":{"architecture":"arm64","boot_time":"2023-03-31T21:44:05.850687-07:00","name":"Marcs-MacBook-Pro-2.local","ip":["127.0.0.1/8","::1/128","fe80::1/64","fe80::7c66:fdff:fe68:bc97/64","fe80::7c66:fdff:fe68:bc96/64","fe80::1870:40dc:ff53:d829/64","192.168.68.77/22","fe80::cc31:b7ff:fe76:f227/64","fe80::cc31:b7ff:fe76:f227/64","fe80::4c34:8ff6:a10b:8856/64","fe80::d2f9:1a77:1652:b697/64","fe80::ce81:b1c:bd2c:69e/64","10.211.55.2/24","fe80::3c06:30ff:fe60:3464/64","10.37.129.2/24","fe80::3c06:30ff:fe60:3465/64"],"kernel_version":"22.4.0","mac":["7e:66:fd:68:bc:97","7e:66:fd:68:bc:96","7e:66:fd:68:bc:76","7e:66:fd:68:bc:77","36:88:28:18:df:40","36:88:28:18:df:44","36:88:28:18:df:40","3e:06:30:06:7e:d3","3c:06:30:06:7e:d3","ce:31:b7:76:f2:27","ce:31:b7:76:f2:27","86:cf:e9:60:e8:63","3e:06:30:60:34:64","0a:50:61:6d:13:1d","3e:06:30:60:34:65"],"os":{"type":"macos","family":"darwin","platform":"darwin","name":"macOS","version":"13.3","major":13,"minor":3,"patch":0,"build":"22E252"},"timezone":"PDT","timezone_offset_sec":-25200},"ecs.version":"1.6.0"}}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.149-0700","log.logger":"beat","log.origin":{"file.name":"beatcmd/beat.go","file.line":620},"message":"Process info","service.name":"apm-server","system_info":{"process":{"cwd":"","exe":"./apm-server","name":"","pid":52432,"ppid":0,"start_time":"1969-12-31T16:00:00.000-0800"},"ecs.version":"1.6.0"}}
{"log.level":"warn","@timestamp":"2023-05-09T09:19:54.149-0700","log.origin":{"file.name":"report/setup_other.go","file.line":29},"message":"Metrics not implemented for this OS.","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.149-0700","log.logger":"config","log.origin":{"file.name":"config/agentconfig.go","file.line":80},"message":"using output.elasticsearch for fetching agent config","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.149-0700","log.logger":"beater","log.origin":{"file.name":"beater/http.go","file.line":142},"message":"Listening on: 127.0.0.1:8200","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.149-0700","log.origin":{"file.name":"beatcmd/beat.go","file.line":394},"message":"apm-server started.","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.150-0700","log.logger":"beater","log.origin":{"file.name":"beater/beater.go","file.line":196},"message":"no cgroups detected, falling back to total system memory","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.150-0700","log.logger":"beater","log.origin":{"file.name":"beater/beater.go","file.line":220},"message":"MaxConcurrentDecoders set to 1024 based on 80 percent of 10.0gb of memory","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.150-0700","log.logger":"beater","log.origin":{"file.name":"beater/beater.go","file.line":226},"message":"Transactions.MaxTransactionGroups set to 50000 based on 10.0gb of memory","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.171-0700","log.logger":"beater","log.origin":{"file.name":"beater/beater.go","file.line":232},"message":"Transactions.MaxServices set to 10000 based on 10.0gb of memory","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.171-0700","log.logger":"beater","log.origin":{"file.name":"beater/beater.go","file.line":238},"message":"ServiceTransactions.MaxGroups for service aggregation set to 10000 based on 10.0gb of memory","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.175-0700","log.logger":"beater","log.origin":{"file.name":"beater/waitready.go","file.line":40},"message":"blocking ingestion until all preconditions are satisfied","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.175-0700","log.logger":"beater","log.origin":{"file.name":"beater/beater.go","file.line":755},"message":"docappender.DocumentBufferSize set to 8192 based on 10.0gb of memory","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.175-0700","log.logger":"beater","log.origin":{"file.name":"beater/beater.go","file.line":769},"message":"docappender.MaxRequests set to 25 based on 10.0gb of memory","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.176-0700","log.logger":"beater","log.origin":{"file.name":"apm-server/main.go","file.line":110},"message":"creating transaction metrics aggregation with config: {MaxTransactionGroups:50000 MaxServices:10000 HDRHistogramSignificantFigures:2}","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.189-0700","log.logger":"beater","log.origin":{"file.name":"apm-server/main.go","file.line":128},"message":"creating service destination metrics aggregation with config: {MaxGroups:10000}","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.189-0700","log.logger":"beater","log.origin":{"file.name":"apm-server/main.go","file.line":148},"message":"creating service transaction metrics aggregation with config: {MaxGroups:10000 HDRHistogramSignificantFigures:2}","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.189-0700","log.logger":"beater","log.origin":{"file.name":"apm-server/main.go","file.line":169},"message":"creating service summary metrics aggregation with config: {MaxGroups:10000 HDRHistogramSignificantFigures:2}","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.189-0700","log.logger":"handler","log.origin":{"file.name":"api/mux.go","file.line":139},"message":"Path / added to request handler","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.189-0700","log.logger":"handler","log.origin":{"file.name":"api/mux.go","file.line":139},"message":"Path /config/v1/agents added to request handler","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.189-0700","log.logger":"handler","log.origin":{"file.name":"api/mux.go","file.line":139},"message":"Path /config/v1/rum/agents added to request handler","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.189-0700","log.logger":"handler","log.origin":{"file.name":"api/mux.go","file.line":139},"message":"Path /intake/v2/rum/events added to request handler","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.189-0700","log.logger":"handler","log.origin":{"file.name":"api/mux.go","file.line":139},"message":"Path /intake/v3/rum/events added to request handler","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.189-0700","log.logger":"handler","log.origin":{"file.name":"api/mux.go","file.line":139},"message":"Path /intake/v2/events added to request handler","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.189-0700","log.logger":"handler","log.origin":{"file.name":"api/mux.go","file.line":139},"message":"Path /v1/traces added to request handler","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.189-0700","log.logger":"handler","log.origin":{"file.name":"api/mux.go","file.line":139},"message":"Path /v1/metrics added to request handler","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.190-0700","log.logger":"handler","log.origin":{"file.name":"api/mux.go","file.line":139},"message":"Path /v1/logs added to request handler","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.190-0700","log.logger":"handler","log.origin":{"file.name":"api/mux.go","file.line":144},"message":"Path /debug/vars added to request handler","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.190-0700","log.logger":"beater","log.origin":{"file.name":"beater/server.go","file.line":208},"message":"Starting apm-server [3503f66af13c5ef1c4439a37584aacc1751464c5 built 2023-05-09 15:18:59 +0000 +0000]. Hit CTRL-C to stop it.","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.190-0700","log.logger":"beater","log.origin":{"file.name":"beater/http.go","file.line":94},"message":"RUM endpoints disabled.","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:19:54.190-0700","log.logger":"beater","log.origin":{"file.name":"beater/http.go","file.line":104},"message":"SSL disabled.","service.name":"apm-server","ecs.version":"1.6.0"}

</p>
</details> 

@kruskall AWS doesn't allow launching macOS Catalina anymore.

image

GitHub only allows launching macOS 11, which is Big Sur (the one after Catalina).

Do you have any suggestions on how to test this? I can't launch a VM with Catalina either since I've got a Mac M1 and Catalina doesn't support ARM architecture.

**Comment by marclop** [comment]
I've also modified the source code to issue the warning for the `22.` prefix, and I've verified that it does log that warning.

```diff
diff --git a/internal/beatcmd/beat.go b/internal/beatcmd/beat.go
index 2ce87c017..8974b3d4e 100644
--- a/internal/beatcmd/beat.go
+++ b/internal/beatcmd/beat.go
@@ -270,7 +270,7 @@ func (b *Beat) Run(ctx context.Context) error {
 	if runtime.GOOS == "darwin" {
 		if host, err := sysinfo.Host(); err != nil {
 			logger.Warnf("failed to retrieve kernel version, ignoring potential deprecation warning: %v", err)
-		} else if strings.HasPrefix(host.Info().KernelVersion, "19.") {
+		} else if strings.HasPrefix(host.Info().KernelVersion, "22.") {
 			// macOS 10.15.x (catalina) means darwin kernel 19.y
 			logger.Warn("deprecation notice: support for macOS 10.15 will be removed in an upcoming version")
 		}

```

<details><summary>Details</summary>
<p>

```
./apm-server -E output.elasticsearch.username=admin -E output.elasticsearch.password=changeme -E apm-server.expvar.enabled=true -d '*' -e
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.343-0700","log.origin":{"file.name":"beatcmd/beat.go","file.line":139},"message":"Home path: [/Users/marclop/repos/elastic/apm-server] Config path: [/Users/marclop/repos/elastic/apm-server] Data path: [/Users/marclop/repos/elastic/apm-server/data] Logs path: [/Users/marclop/repos/elastic/apm-server/logs]","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"debug","@timestamp":"2023-05-09T09:27:06.343-0700","log.logger":"beat","log.origin":{"file.name":"beatcmd/beat.go","file.line":172},"message":"Beat metadata path: /Users/marclop/repos/elastic/apm-server/data/meta.json","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.343-0700","log.origin":{"file.name":"beatcmd/beat.go","file.line":146},"message":"Beat ID: bf2a6d26-c5ed-4b41-a507-feab5b75c095","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"warn","@timestamp":"2023-05-09T09:27:06.344-0700","log.origin":{"file.name":"beatcmd/beat.go","file.line":275},"message":"deprecation notice: support for macOS 10.15 will be removed in an upcoming version","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.344-0700","log.logger":"beat","log.origin":{"file.name":"beatcmd/beat.go","file.line":576},"message":"Beat info","service.name":"apm-server","system_info":{"beat":{"path":{"config":"/Users/marclop/repos/elastic/apm-server","data":"/Users/marclop/repos/elastic/apm-server/data","home":"/Users/marclop/repos/elastic/apm-server","logs":"/Users/marclop/repos/elastic/apm-server/logs"},"type":"apm-server","uuid":"bf2a6d26-c5ed-4b41-a507-feab5b75c095"},"ecs.version":"1.6.0"}}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.344-0700","log.logger":"beat","log.origin":{"file.name":"beatcmd/beat.go","file.line":584},"message":"Build info","service.name":"apm-server","system_info":{"build":{"commit":"3503f66af13c5ef1c4439a37584aacc1751464c5","time":"2023-05-09T15:18:59.000Z","version":"8.9.0"},"ecs.version":"1.6.0"}}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.344-0700","log.logger":"beat","log.origin":{"file.name":"beatcmd/beat.go","file.line":587},"message":"Go runtime info","service.name":"apm-server","system_info":{"go":{"os":"darwin","arch":"arm64","max_procs":8,"version":"go1.20.4"},"ecs.version":"1.6.0"}}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.344-0700","log.origin":{"file.name":"beatcmd/maxprocs.go","file.line":68},"message":"maxprocs: Leaving GOMAXPROCS=8: CPU quota undefined","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.345-0700","log.logger":"beat","log.origin":{"file.name":"beatcmd/beat.go","file.line":591},"message":"Host info","service.name":"apm-server","system_info":{"host":{"architecture":"arm64","boot_time":"2023-03-31T21:44:05.850687-07:00","name":"Marcs-MacBook-Pro-2.local","ip":["127.0.0.1/8","::1/128","fe80::1/64","fe80::7c66:fdff:fe68:bc97/64","fe80::7c66:fdff:fe68:bc96/64","fe80::1870:40dc:ff53:d829/64","192.168.68.77/22","fe80::5cee:d4ff:fe2f:df4/64","fe80::5cee:d4ff:fe2f:df4/64","fe80::4c34:8ff6:a10b:8856/64","fe80::d2f9:1a77:1652:b697/64","fe80::ce81:b1c:bd2c:69e/64","10.211.55.2/24","fe80::3c06:30ff:fe60:3464/64","10.37.129.2/24","fe80::3c06:30ff:fe60:3465/64"],"kernel_version":"22.4.0","mac":["7e:66:fd:68:bc:97","7e:66:fd:68:bc:96","7e:66:fd:68:bc:76","7e:66:fd:68:bc:77","36:88:28:18:df:40","36:88:28:18:df:44","36:88:28:18:df:40","3e:06:30:06:7e:d3","3c:06:30:06:7e:d3","5e:ee:d4:2f:0d:f4","5e:ee:d4:2f:0d:f4","86:cf:e9:60:e8:63","3e:06:30:60:34:64","0a:50:61:6d:13:1d","3e:06:30:60:34:65"],"os":{"type":"macos","family":"darwin","platform":"darwin","name":"macOS","version":"13.3","major":13,"minor":3,"patch":0,"build":"22E252"},"timezone":"PDT","timezone_offset_sec":-25200},"ecs.version":"1.6.0"}}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.345-0700","log.logger":"beat","log.origin":{"file.name":"beatcmd/beat.go","file.line":620},"message":"Process info","service.name":"apm-server","system_info":{"process":{"cwd":"","exe":"./apm-server","name":"","pid":55131,"ppid":0,"start_time":"1969-12-31T16:00:00.000-0800"},"ecs.version":"1.6.0"}}
{"log.level":"warn","@timestamp":"2023-05-09T09:27:06.345-0700","log.origin":{"file.name":"report/setup_other.go","file.line":29},"message":"Metrics not implemented for this OS.","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.345-0700","log.logger":"config","log.origin":{"file.name":"config/agentconfig.go","file.line":80},"message":"using output.elasticsearch for fetching agent config","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.345-0700","log.logger":"beater","log.origin":{"file.name":"beater/http.go","file.line":142},"message":"Listening on: 127.0.0.1:8200","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.345-0700","log.origin":{"file.name":"beatcmd/beat.go","file.line":394},"message":"apm-server started.","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.346-0700","log.logger":"beater","log.origin":{"file.name":"beater/beater.go","file.line":196},"message":"no cgroups detected, falling back to total system memory","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.346-0700","log.logger":"beater","log.origin":{"file.name":"beater/beater.go","file.line":220},"message":"MaxConcurrentDecoders set to 1024 based on 80 percent of 10.0gb of memory","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.346-0700","log.logger":"beater","log.origin":{"file.name":"beater/beater.go","file.line":226},"message":"Transactions.MaxTransactionGroups set to 50000 based on 10.0gb of memory","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.346-0700","log.logger":"beater","log.origin":{"file.name":"beater/beater.go","file.line":232},"message":"Transactions.MaxServices set to 10000 based on 10.0gb of memory","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.346-0700","log.logger":"beater","log.origin":{"file.name":"beater/beater.go","file.line":238},"message":"ServiceTransactions.MaxGroups for service aggregation set to 10000 based on 10.0gb of memory","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.350-0700","log.logger":"beater","log.origin":{"file.name":"beater/beater.go","file.line":755},"message":"docappender.DocumentBufferSize set to 8192 based on 10.0gb of memory","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.350-0700","log.logger":"beater","log.origin":{"file.name":"beater/beater.go","file.line":769},"message":"docappender.MaxRequests set to 25 based on 10.0gb of memory","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.350-0700","log.logger":"beater","log.origin":{"file.name":"beater/waitready.go","file.line":40},"message":"blocking ingestion until all preconditions are satisfied","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.351-0700","log.logger":"beater","log.origin":{"file.name":"apm-server/main.go","file.line":110},"message":"creating transaction metrics aggregation with config: {MaxTransactionGroups:50000 MaxServices:10000 HDRHistogramSignificantFigures:2}","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.414-0700","log.logger":"beater","log.origin":{"file.name":"apm-server/main.go","file.line":128},"message":"creating service destination metrics aggregation with config: {MaxGroups:10000}","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.414-0700","log.logger":"beater","log.origin":{"file.name":"apm-server/main.go","file.line":148},"message":"creating service transaction metrics aggregation with config: {MaxGroups:10000 HDRHistogramSignificantFigures:2}","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.414-0700","log.logger":"beater","log.origin":{"file.name":"apm-server/main.go","file.line":169},"message":"creating service summary metrics aggregation with config: {MaxGroups:10000 HDRHistogramSignificantFigures:2}","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.415-0700","log.logger":"handler","log.origin":{"file.name":"api/mux.go","file.line":139},"message":"Path / added to request handler","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.415-0700","log.logger":"handler","log.origin":{"file.name":"api/mux.go","file.line":139},"message":"Path /config/v1/agents added to request handler","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.415-0700","log.logger":"handler","log.origin":{"file.name":"api/mux.go","file.line":139},"message":"Path /config/v1/rum/agents added to request handler","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.415-0700","log.logger":"handler","log.origin":{"file.name":"api/mux.go","file.line":139},"message":"Path /intake/v2/rum/events added to request handler","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.415-0700","log.logger":"handler","log.origin":{"file.name":"api/mux.go","file.line":139},"message":"Path /intake/v3/rum/events added to request handler","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.415-0700","log.logger":"handler","log.origin":{"file.name":"api/mux.go","file.line":139},"message":"Path /intake/v2/events added to request handler","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.415-0700","log.logger":"handler","log.origin":{"file.name":"api/mux.go","file.line":139},"message":"Path /v1/traces added to request handler","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.415-0700","log.logger":"handler","log.origin":{"file.name":"api/mux.go","file.line":139},"message":"Path /v1/metrics added to request handler","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.415-0700","log.logger":"handler","log.origin":{"file.name":"api/mux.go","file.line":139},"message":"Path /v1/logs added to request handler","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.415-0700","log.logger":"handler","log.origin":{"file.name":"api/mux.go","file.line":144},"message":"Path /debug/vars added to request handler","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.415-0700","log.logger":"beater","log.origin":{"file.name":"beater/server.go","file.line":208},"message":"Starting apm-server [3503f66af13c5ef1c4439a37584aacc1751464c5 built 2023-05-09 15:18:59 +0000 +0000]. Hit CTRL-C to stop it.","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.415-0700","log.logger":"beater","log.origin":{"file.name":"beater/http.go","file.line":94},"message":"RUM endpoints disabled.","service.name":"apm-server","ecs.version":"1.6.0"}
{"log.level":"info","@timestamp":"2023-05-09T09:27:06.415-0700","log.logger":"beater","log.origin":{"file.name":"beater/http.go","file.line":104},"message":"SSL disabled.","service.name":"apm-server","ecs.version":"1.6.0"}
```

</p>
</details>

**Pull request merged**